charts/internal/calico/templates/kube-controller/deployment-calico-kube-controllers.yaml

```diff
@@ -57,6 +57,8 @@ spec:
               memory: 50Mi
             limits:
               memory: 2Gi
+          securityContext:
+            allowPrivilegeEscalation: false
```

Comment:
There is a defined `securityContext` in this container. Please add the change there.